app/src/pages/inside/common/testCaseList/utils.ts

```diff
@@ -89,3 +89,15 @@ export function buildBreadcrumbs(folders: Folder[], folderId: number): string[]
 
   return [folder.name];
 }
+
+export const formatTimestampForSidePanel = (timestamp: string): string => {
+  if (!timestamp) return '';
+  const date = new Date(timestamp);
+  return date.toLocaleString('en-US', {
```

Comment:
```suggestion
  if (!timestamp) { 
    return '';
  };

  return new Date(timestamp).toLocaleString('en-US', {
```

I see that we have already `date-fns` can achieve the same with it?